package.json

```diff
@@ -8,19 +8,19 @@
 		"setup": "rm -r dist && npm ci && npm run export-version && npm run build:only --production && npm run baseline && npm run prisma:migrate && npm run commands:deploy",
 		"commands:deploy": "node . --deploy",
 		"commands:revoke": "node . --revoke",
-		"baseline": "./node_modules/.bin/prisma migrate resolve --applied 20220915022732_initial_state  # TODO: Use this to set initial tables on the database",
-		"prisma:migrate": "./node_modules/.bin/prisma migrate deploy",
-		"prisma:introspect": "./node_modules/.bin/prisma db pull",
-		"prisma:generate": "./node_modules/.bin/prisma generate --schema ./prisma/schema.prisma",
-		"migrations:generate": "./node_modules/.bin/prisma migrate dev --create-only",
+		"baseline": "prisma migrate resolve --applied 20220915022732_initial_state  # TODO: Use this to set initial tables on the database",
+		"prisma:migrate": "prisma migrate deploy",
+		"prisma:introspect": "prisma db pull",
+		"prisma:generate": "prisma generate --schema ./prisma/schema.prisma",
+		"migrations:generate": "prisma migrate dev --create-only",
 		"lint": "npm run export-version && npm run lintonly",
 		"lint:fix": "npm run export-version && npm run lintonly -- --fix",
-		"lintonly": "./node_modules/.bin/eslint . --ext .ts",
+		"lintonly": "eslint . --ext .ts",
 		"prebuild": "npm run lint",
 		"build": "npm run build:only",
 		"build:only": "rm -rf dist && ./node_modules/.bin/tsc -p tsconfig.prod.json",
-		"export-version": "./node_modules/.bin/genversion ./src/constants/version.ts -es",
-		"test": "./node_modules/.bin/jest",
+		"export-version": "genversion ./src/constants/version.ts -es",
+		"test": "jest",
```

Comment:
These scripts pointed to specific binaries on purpose, to disambiguate which version of CLI tools we're using.

For example, Prisma's CLI recently had an update (v4.4.0) which breaks the way index sorting is defined in the schema. (I haven't found an open issue on this, but this is a real difference in behavior that I've observed in other projects.) For this reason, we should use the known-good version (v4.3.1) as specified in our `package.json` and `package-lock.json` files. If we instead install the latest version of the CLI globally, and the NPM scripts are not specific, then that bugged arbitrary version will be used instead of the specific known-good in the project's `node_modules` directory.

This principle of specificity applies not just to Prisma's CLI, but all of them. Some of them ask that the CLI be installed globally, but in the interest of specificity, I've opted to install all of them as `devDependencies` and reference their binaries in the scripts.